reframe/frontend/executors/policies.py

```diff
@@ -17,6 +18,10 @@
                                         TaskEventListener, ABORT_REASONS)
 
 
+def values_len(d):
```

Comment:
The name is too generic for being in the global scope. Use something a bit more specific, for example:

```suggestion
def dictlist_len(d):
```